lib/main.dart

```diff
@@ -219,7 +219,7 @@ class AppState extends ConsumerState<App> with WidgetsBindingObserver {
           title: 'Archethic Wallet',
           theme: ThemeData(
             dialogBackgroundColor: theme.backgroundDark,
-            primaryColor: theme.text,
+            colorSchemeSeed: theme.text,
```

Comment:
In material 3, widgets need to have a colorSchemeSeed instead of a primary color (for example when we tap on a button (long press) we will see a color inside => this refers to the colorSchemeSeed).
Default value for that is light purple.